documentation/secure_storage.md

```diff
@@ -159,6 +159,18 @@ to encrypt/decrypt the FEK.
 TSK is derived by:
 > TSK = HMAC<sub>SHA256</sub> (SSK, TA_UUID)
 
+#### TA storage space isolation
+
+OP-TEE provides different folders for different TAs in Linux file system for
+storing their own TEE files, but OP-TEE cannot prevent an attacker from
+directly copying a TEE file from one TA's folder to another TA's folder in
+Linux file system. TEE OS should have the ability to detect those kind of
```

Comment:
With TSK added I believe the part here saying `"TEE OS should have the .... meet the requirement"` should be removed also?